ingest/api/ingestapi.py

```diff
@@ -443,10 +443,8 @@ def link_entity(self, from_entity, to_entity, relationship, is_collection=True):
 
         self.logger.info('fromUri ' + from_uri + ' toUri:' + to_uri)
 
-        headers = {
-            'Content-type': 'text/uri-list',
-            'Authorization': self.get_headers()['Authorization']
-        }
+        headers = self.get_headers()
+        headers['Content-type'] = 'text/uri-list'
```

Comment:
It might be that other scripts are using ingest_api, not just the importer. It's better to only add the auth header when the token is not present

Reply:
That's what this does?

Reply:
ah i meant, what  would happen if ingestapi.py is used in a standalone script and it needs to link entities?

Reply:
My change doesn't remove the authorization token. It just allows everything to proceed if it's not there

Reply:
oh you're right! you're just overriding the content type. Somehow I thought you were removing the auth header, but you're not.